include/projectile.h

```diff
@@ -5,8 +5,8 @@
 
 class Projectile : public Moveable {
     private:
-        const float move_speed = 100.f;
-        const float max_distance = 500.f;
+        const float move_speed = 400.f;
```

Comment:
This seems like the projectiles are going to be moving really fast, I assume that you have tested it though? 150 also seems like a really short distance before the projectile disappears.